cpu/efm32/efm32-vars.mk

```diff
@@ -0,0 +1,139 @@
+CPU_FAM_INDEX           = 1
```

Comment:
BTW @basilfx I didn't keep the message that was in `cpu.txt`:

```
# This file is automatically generated, and should not be changed. There is
# probably little reason to edit this file anyway, since it should already
# contain all information for the EFM32GG family of CPUs.
```

since this wasn't generated automatically but I did copy paste everything so not data was changed.